docs/modules/ROOT/pages/push.adoc

```diff
@@ -0,0 +1,94 @@
+== AeroGear JavaScript Unifiedpush SDK
+
+An SDK that allows you to receive push notifications and integrate with the Aerogear Unifiedpush Server. In contrast to the other SDKs, `@aerogear/push` itself does not implement the handling of push notifications. It helps you to integrate a third party push library with our backend, the Aerogear Unifiedpush Server. While you can use any push plugin you like that supports Firebase Cloud Messaging and Apple Push Notification Services we suggest link:https://github.com/phonegap/phonegap-plugin-push[Phonegap Plugin Push] for Cordova and Ionic Apps.
+
+=== Installation
+
+For a Cordova App two packages need to be installed: `@aerogear/push` and the platform push plugin:
+
+[source,bash]
+----
+$ npm install --save @aerogear/push
+$ cordova plugin add phonegap-plugin-push
+----
+
+Ionic Apps require an additional dependency, the link:https://ionicframework.com/docs/native/push/[Ionic Native Push Library]
+
+[source,bash]
+----
+npm install --save @ionic-native/push
+----
+
+=== Configuration
+
+Make sure that your `mobile-serivces.json` file has a configuration object for push. If not you can retrieve it from the mobile client UI in Openshift or using the mobile cli:
+
+[source,bash]
+----
+$ mobile get clientconfig <mobile client id> --namespace=<namespace> -o json
+----
+
+Copy the over the `push` part of the configuration:
+
+----
+{
+  ...
+  "services": [
+    {
+      "id": "push",
+      "name": "push",
+      "type": "push",
+      "url": <backend url>,
+      "config": {
+        "ios":      <iOS specific config>
+        "android":  <Android specific config>
+      }
+    }
+  ]
+}
+----
+
+Next you need to configure the push provider itself. `phonegap-plugin-push` defaults to Firebase Cloud Messaging on Android and iOS. Login to your link:https://firebase.google.com/[Firebase account] and retrieve the config file (`google-services.json` or `google-services.plist`). Include it in the root of your project folder. Please refer to the link:https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/INSTALLATION.md#android-details[Plugin documentation] for more details.
+
+=== Usage
+
+To make use of the Aerogear Unifiedpush Server, devices need to be registered with it. This in handled by `@aerogear/push` and should be performed after the platform push plugin itself has finished initialization:
```

Comment:
`To make use of the Aerogear Unifiedpush Server, devices need to be registered with it` => Unified Push Server allows sending messages to all registered devices, independent of the platform that is used.